ComfortZoneApp/app/src/main/java/com/example/comfortzone/notification/NotificationHelper.java

```diff
@@ -16,6 +16,7 @@
 public class NotificationHelper {
 
     private Context context;
+    public static final String ARG_IS_FROM_NOTIF = "FRAGMENT";
```

Comment:
as an exercise, to make it more scalable, you should use a string extra, where you pass in which screen to open (i.e. AUTO_OPEN_SCREEN)

then you will check if it's flight, input, profile, and define a default screen/stay at current screen